docs/source/getting_started/basic_functionality.rst

```diff
@@ -67,9 +67,19 @@ There are currently three ways to filter, and they only work on data model objec
 :func:`~citrine.resources.data_concepts.DataConceptsCollection.filter_by_tags`,
 :func:`~citrine.resources.data_concepts.DataConceptsCollection.filter_by_attribute_bounds`,
 and :func:`~citrine.resources.data_concepts.DataConceptsCollection.filter_by_name`.
+
 Filtering by tags or attribute bounds can be scoped to a dataset or to a project.
 Filtering by name must be scoped to a dataset.
 
+Get Material History
+^^^^^^^^^^^^^^^^^^^^
+
+Starting with a specific root :class:`MaterialRun <citrine.resources.material_run.MaterialRun>`,
+you can retrieve the complete material history--every process, ingredient, material, and measurement
+that that went into the root material and that you have read access to. The method is
```

Comment:
every process, ingredient and material that went into the root material .., as well as the measurements that were performed on all of those materials

?

Reply:
Good suggestion, thanks